Gemfile.lock

```diff
@@ -8,86 +8,65 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (7.0.6)
```

Comment:
Please rollback the bundle update. Shouldn't be necessary for this change and it obscures the PR.